testutil/beaconmock/options.go

```diff
@@ -367,6 +367,37 @@ func WithNoAttesterDuties() Option {
 	}
 }
 
+func WithSyncCommitteeDuties() Option {
```

Comment:
add a comment what this does. Suggest just turning this duty on always if called?

Reply:
for simnet?
like adding it in app.go at the time of instantiating beaconmock